cli/src/commands/shell/index.ts

```diff
@@ -111,6 +120,11 @@ export default class Shell extends BaseCommand {
 
     replServer.context.xata = new XataClient({ fetch, apiKey });
     replServer.context.api = new XataApiClient({ fetch, apiKey });
+    replServer.context.api.GET = (path: string) => fetchApi('GET', path);
+    replServer.context.api.POST = (path: string, body?: any) => fetchApi('POST', path, JSON.stringify(body));
+    replServer.context.api.PATCH = (path: string, body?: any) => fetchApi('PATCH', path, JSON.stringify(body));
+    replServer.context.api.PUT = (path: string, body?: any) => fetchApi('PUT', path, JSON.stringify(body));
+    replServer.context.api.DELETE = (path: string) => fetchApi('DELETE', path);
```

Comment:
Nice! Thanks :)

Would it be possible to do the same in the SDK playground?

Reply:
Then I'd move it to another place